src/store/store.js

```diff
@@ -0,0 +1,8 @@
+import { createStore, applyMiddleware } from "redux";
+import thunkMiddleware from "redux-thunk";
+import { movieReducer } from "./reducers";
+
+export const store = createStore(
+  movieReducer,
+  applyMiddleware(thunkMiddleware)
+);
```

Comment:
It would be wonderful if you could add [redux-devtools](https://github.com/zalmoxisus/redux-devtools-extension).
In the future, it could save you more than one hour. ⏳